app/src/main/java/com/ataulm/artcollector/paintings/data/AndroidPaintingsRepository.kt

```diff
@@ -12,15 +14,24 @@ internal class AndroidPaintingsRepository @Inject constructor(
 
     override suspend fun paintings(): List<Painting> {
         return harvardArtMuseumApi.paintings().await()
-                .records.map { it.toPainting() }
+                .records
+                .filter { it.people?.find(whereApiPersonIsAKnownArtist) != null }
```

Comment:
would be cool to do this from the API but I don't think so - I opened an issue here: https://github.com/harvardartmuseums/api-docs/issues/5